docker/roles/generate_containers/tasks/main.yml

```diff
@@ -16,46 +16,32 @@
     src: ./src/
     dest: ./{{ base_os }}/
 
-- name: get stat for templates (ansible 1.x)
-  stat:
-    path: "{{ item }}"
-  with_fileglob: "../../../{{ base_os }}/*.j2"
-  register: stat_templates
-  when: ansible_version.major == 1
-
-
-- name: save stat output
-  set_fact:
-    template_list: "{{ stat_templates }}"
-  when: not stat_templates|skipped
-
-- name: get stat for templates (ansible 2.x)
-  stat:
-    path: "{{ item }}"
-  with_fileglob: "{{ base_os }}/*.j2"
-  register: stat_templates
-  when: ansible_version.major > 1
+- name: find templates
+  find:
+    paths: '{{ base_os }}/'
+    patterns: '*.j2'
+    recurse: yes
+  register: find_templates
```

Comment:
Why not just register this task as 'template_list' and remove the 'set_fact' that follows?

Other than this one thing LGTM.

Reply:
Good question... I'll update the PR

Reply:
@joelddiaz fixed